### PR TITLE
Bug 1918757: added document ready state to fix this page loading issue

### PR DESCRIPTION
**Pull request by makambalaji**
This pr should fix the flakiness of project creation feature file

### CODE REVIEW

File: frontend/packages/dev-console/integration-tests/support/pages/app.ts [hunk 1]
```diff
@@ -109,11 +109,15 @@ export const naviagteTo = (opt: devNavigationMenu) => {
...
-    cy.byLegacyTestID('namespace-bar-dropdown')
-      .find('button')
-      .eq(0)
-      .click();
-    cy.byTestDropDownMenu('#CREATE_RESOURCE_ACTION#').click();
+    cy.document().then((doc) => {
+      if (doc.readyState === 'complete') {
+        cy.byLegacyTestID('namespace-bar-dropdown')
+          .find('button')
+          .eq(0)
```
Comment (dtaylor113) [inline frontend/packages/dev-console/integration-tests/support/pages/app.ts:116]: could be `.first()`

Comment (makambalaji) [reply frontend/packages/dev-console/integration-tests/support/pages/app.ts:116]: Both works, it doesn't make any difference right?

**Comment by jerolimov** [comment]
> terraform issue The number of members in the policy (1,501) is larger than the maximum allowed size 1,500.

We will need to run/retest this later again

File: frontend/packages/dev-console/integration-tests/support/pages/app.ts (hunk 1 above)
Comment (dtaylor113) [reply frontend/packages/dev-console/integration-tests/support/pages/app.ts:116]: just readability :-)

**Review by dtaylor113 — APPROVED** [review]
/lgtm

**Pull request merged**